src/util/time.rs

```diff
@@ -0,0 +1,10 @@
+//! Util time conversion functions
+use std::time::Duration;
+///Converts a Duration to the time in seconds.
+pub fn duration_to_secs(duration: &Duration) -> f32 {
```

Comment:
Stylistically, a copy would be preferable if the type is `Copy`.

Reply:
Fixed, thanks for the tip, I wasn't aware that Duration was Copy